apps/swap-service/src/swaps/utils.ts

```
@@ -61,6 +62,35 @@ export const resolveStalledSwap = (
+      return { status: 'PENDING', details: { checked: false, reason: 'unsupported-chain' } }
+    case 'not-found':
+      return { status: 'PENDING', details: { checked: false, reason: 'tx-not-found' } }
+    case 'error':
+      return { status: 'PENDING', details: { checked: false, reason: 'lookup-failed' } }
+    case 'found': {
+      // unchained reports seconds
+      const blockTime = lookup.timestamp * 1000
+      const quoted = quotedAt.getTime()
+      const checked = { checked: true, blockTime, quotedAt: quoted }
+
+      if (quoted <= blockTime + toleranceMs) return { status: 'ACCEPTED', details: { ...checked, reason: 'quote-precedes-tx' } }
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)